src/app/content-right/report/report.component.html

```diff
@@ -76,6 +76,7 @@ <h4>{{data.title}}</h4>
 						</table>
 					</div>
 					<!-- Fire Hydrology: Query by Fire Perimeters-->
+					<div *ngIf="output.key=='downstreamDist'">Downstream Trace Distance (miles): {{output.value}}</div>
```

Comment:
@Anders-Hopkins Do you know if the `downstream_dist` in the nldi_polygon_query is miles or something else? 
https://streamstats.usgs.gov/nldipolygonservices/docs#/default/query_poly_nldi_poly_query__post

Reply:
@harper-wavra Distance is in kilometers: https://waterdata.usgs.gov/blog/nldi-intro/#:~:text=The%20Hydro%20Network%2DLinked%20Data%20Index%20(NLDI)%20is%20a,and%20outlets%20of%20HUC12%20watersheds.